code/script.js

```diff
@@ -1,19 +1,164 @@
+} else if (foodChoice === "2") {
+  selectedFoodChoice = "Pasta";
+  alert(`You've chosen ${selectedFoodChoice}!`);
+} else if (foodChoice === "3") {
+  selectedFoodChoice = "Salad";
+  alert(`You've chosen ${selectedFoodChoice}!`);
+} else {
+  alert("Invaild choice. Please select a number between 1 and 3.");
+}
+
+
+// Step 3 - Subtype food choice
+let pizzaType = ""
+let pastaType = ""
+let saladType = ""
+let subtypeChoice = ""
+
+if (selectedFoodChoice === "Pizza") {
+  pizzaType = prompt(
+    `Delicious! What kind of pizza do you want? Type a number.
+    1 - Napolitiana
+    2 - Hawaiian
+    3 - Pepperoni`
+  );
+  if (pizzaType == 1) {
+    subtypeChoice = "Napolitiana"
+    alert(
+      `You chose a ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else if (pizzaType == 2) {
+    subtypeChoice = "Hawaiian"
+    alert(
+      `You chose a ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else if (pizzaType == 3) {
+    subtypeChoice = "Pepperoni"
+    alert(
+      `You chose a ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else {
+    alert("Invalid choice. Please select a number between 1 and 3.");
+  }
+}
+
+if (selectedFoodChoice === "Pasta") {
+  pastaType = prompt(
+    `Delicious! What kind of pasta do you want? Type a number.
+    1 - Spaghetti Carbonara
+    2 - Fettuccine Alfredo
+    3 - Cheesy Tortellini`
+  );
+  if (pastaType == 1) {
+    subtypeChoice = "Spaghetti Carbonara"
+    alert(
+      `You chose ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else if (pastaType == 2) {
+    subtypeChoice = "Fettuccine Alfredo"
+    alert(
+      `You chose ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else if (pastaType == 3) {
+    subtypeChoice = "Cheesy Tortellini"
+    alert(
+      `You chose ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else {
+    alert("Invalid choice. Please select a number between 1 and 3.");
+  }
+}
 
-// Step 3 - Subtype choice
-// Your code goes here
+if (selectedFoodChoice === "Salad") {
+  saladType = prompt(
+    `Delicious! What kind of salad do you want? Type a number.
+    1 - Caesar 
+    2 - Caprese 
+    3 - Greek`
+  );
+  if (saladType == 1) {
+    subtypeChoice = "Caesar"
+    alert(
+      `You chose ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else if (saladType == 2) {
+    subtypeChoice = "Caprese"
+    alert(
+      `You chose ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else if (saladType == 3) {
+    subtypeChoice = "Greek"
+    alert(
+      `You chose ${subtypeChoice} ${selectedFoodChoice}.`
+    )
+  } else {
+    alert("Invalid choice. Please select a number between 1 and 3.");
+  }
+}
+
+
+// Step 4 - Age 
+// Price for regular sizes for adults
+let foodCost = "";
+switch (selectedFoodChoice) {
+  case "Pizza":
+    price = "120";
+    break;
+  case "Pasta":
+    price = "110";
+    break;
+  case "Salad":
+    price = "100";
+    break;
+}
+// Cost for children is half the price
+let foodCostChild = price/2;
```

Comment:
Really liked how you solved this with half the price! Looks clean!

Reply:
Thank you!!